itests/hive-blobstore/src/test/queries/clientpositive/having.q

```diff
@@ -1,4 +1,5 @@
 -- Test HAVING clause
```

Comment:
Since we are supporting this through CBO, I think it'd be nice to include a CBO plan for one of the queries below.
Also, it would be good to add a more complex test with multiple column aliases in the having clause.

Reply:
Adding more tests and EXPLAIN CBO plans is a good idea.